mlperf_logging/compliance_checker/training_4.0.0/closed_gnn.yaml

```diff
@@ -0,0 +1,21 @@
+- KEY:
+    NAME:  global_batch_size
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] > 0"
+
+- KEY:
+    NAME:  opt_name
```

Comment:
In the reference code, the optimizer name is not reported, but we do have a `SEED` exported at [this line](https://github.com/mlcommons/training/blob/3afa89d9094e975aa77b3e1e2af41a2e0af9b0ed/graph_neural_network/train_rgnn_multi_gpu.py#L317). Should we report the optimizer name as well in the reference code? And should we also check if the seed is included in the log as well?

Reply:
We should report optimizer name in reference. The seed check is part of package_checker and we don't need to include it here.

Reply:
Added optimizer name in reference.